flutter-candidate.txt

```diff
@@ -1 +1 @@
-7313c873463d2a20a7ec1047d7b8b9146dd2a39b
+de872dd864518d436062354052ea8604c38e8c94
```

Comment:
is this intentional? @elliette just submitted a PR this morning that updated this to the latest

Reply:
Yes, this is intentional. `7313c873463d2a20a7ec1047d7b8b9146dd2a39b` points to a change from two days ago that doesn't include the `vm_service` version bump.